sharing_configs/client_util.py

```diff
@@ -58,13 +59,19 @@ def import_data(self, folder: str, filename: str) -> bytes:
             raise ApiException("Error during import of object")
         return resp.content
 
-    def get_folders(self, permission: dict) -> dict:
+    def get_folders(self, permission: Union[str, None]) -> dict:
```

Comment:
```suggestion
    def get_folders(self, permission: Optional[str]) -> dict:
```

Could you please change it everywhere with `Union[str, None]` ?

Reply:
Done